packages/1ui/src/components/NetworkStats/NetworkStats.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { useGetStatsQuery } from '../../../../graphql/dist'
+import { NetworkStats } from './NetworkStats'
+
+const meta: Meta<typeof NetworkStats> = {
+  title: 'Components/Network/NetworkStats',
+  component: NetworkStats,
+  parameters: {
+    layout: 'fullscreen',
+  },
+}
+
+export default meta
+
+type Story = StoryObj<typeof NetworkStats>
+
+export const Default: Story = {
+  decorators: [
+    (Story) => (
+      <div style={{ minWidth: '1200px' }}>
+        <Story />
+      </div>
+    ),
+  ],
+  args: {
+    tvl: 13.34,
+    atomsCount: 4200,
+    triplesCount: 4200,
+    signalsCount: 4200,
+    usersCount: 4200,
+  },
+}
+
+const SmartNetworkStats = () => {
+  const { data: systemStats } = useGetStatsQuery(
```

Comment:
Ahh if you were having issues with this query in the component we may need to do some additional config at the decorator level to support the new schema! It'll work fine in launchpad -- will look into this if you were still seeing it.